notes/20200217_meeting.md

```diff
@@ -33,11 +34,15 @@ Other benefits of RAP include:
 ### Improved quality: 
 - no manual steps (in the part of the process selected for RAP)
 - peer review of code
-- testing (*unclear how prescreptive we want to be here*)
+- testing (*unclear how prescriptive we want to be here*)
 
 ### Increased trust:
+- code and documentation open to anyone (if possible)
+- software open to anyone?
 
 ### More efficient processes:
```

Comment:
Would evidence of time saved fit under here? We collect time taken for the process pre and post RAP and use that to demonstrate efficency savings.

Reply:
Yes good idea

Reply:
@ciarag01 can I see that data?

Reply:
I believe that there will be a presentation about this at the RAP meet up (Ciara can correct me if that's wrong!). I also think this data might be provisional just now, but I'm sure it can be shared in due course.

Reply:
Just checked and this will be presented at the meetup! @alexander-newton I can still send this on to you if you'd like to see it sooner?